# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/patches/pkg_binutils.patch
# Copyright (C) 2004 - 2011 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Some quirks needed for compilation with dietlibc.

  - Rene Rebe <rene@exactcode.de>

--- ./libiberty/regex.c~	Fri Sep 27 17:33:09 2002
+++ ./libiberty/regex.c	Sun Dec 29 04:47:22 2002
@@ -302,7 +302,7 @@
 
 /* Integer type for pointers.  */
 # if !defined _LIBC && !defined HAVE_UINTPTR_T
-typedef unsigned long int uintptr_t;
+/*typedef unsigned long int uintptr_t;*/
 # endif
 
 /* Should we use malloc or alloca?  If REGEX_MALLOC is not defined, we
--- binutils-2.16.91.0.3/gprof/gprof.h.vanilla	2005-10-11 17:12:07.000000000 +0200
+++ binutils-2.16.91.0.3/gprof/gprof.h	2005-10-11 17:12:17.000000000 +0200
@@ -136,4 +136,6 @@
 
 extern void done (int status) ATTRIBUTE_NORETURN;
 
+#undef PC
+
 #endif /* gprof_h */
